pkgs/development/libraries/librime/default.nix

```diff
@@ -1,5 +1,5 @@
 { lib, stdenv, fetchFromGitHub, cmake, boost, glog, leveldb, marisa, opencc,
-  yaml-cpp, gtest, capnproto, pkg-config, plugins ? [ ] }:
+  yaml-cpp, gtest, capnproto, pkg-config, librime-lua, plugins ? [ librime-lua ] }:
```

Comment:
Is `librime-lua` really used by most user? Should be it the default plugin?

Reply:
> Is librime-lua really used by most user?

Not sure.  However, the popular `rime-ice` needs `librime-lua`.

Arch Linux also [enables it by default][1].

[1]: https://gitlab.archlinux.org/archlinux/packaging/packages/librime/-/blob/0da73fbd402865409d0f4ef186fb763e9b4523e2/PKGBUILD#L22